lib/core/bytes.nit

```diff
@@ -465,6 +465,7 @@ class Bytes
 	redef fun enlarge(sz) do
 		if capacity >= sz then return
 		persisted = false
+		if capacity < 16 then capacity = 16
```

Comment:
do you have some hard numbers for this?

Reply:
I discussed it a bit in #2549, but I did not give precise numbers. From memory, it saved about 0.1 seconds in the call to nitdoc. It was in the margin of error, but it was the only difference between the implementations of FlatString and Bytes. The effect was visible in the callgrind output, I could retrieve it if you want.

For reference, the resulting run time where it save ~0.1 s:
~~~
$ time nitdoc core -d out
real	0m4.013s
user	0m1.996s
sys	0m0.736s
~~~

Reply:
ok then